db/migrate/20200630120116_create_groups.rb

```diff
@@ -0,0 +1,9 @@
+class CreateGroups < ActiveRecord::Migration[6.0]
+  def change
+    create_table :groups do |t|
+      t.string :class_name,
+
```

Comment:
missing t.references :teachers, I get you didn't do it before we had the teachers, maybe we should have waited for that! so don't forget adding a migration for it once you have our teachers model in you pc

Reply:
we can add the reference when both models are merged to the master